x-pack/plugins/observability/public/pages/cases/case_details.tsx

```diff
@@ -24,19 +24,15 @@ export const CaseDetailsPage = React.memo(() => {
     subCaseId?: string;
   }>();
 
-  if (userPermissions != null && !userPermissions.read) {
-    navigateToApp(`${CASES_APP_ID}`);
-    return null;
-  }
+  useEffect(() => {
```

Comment:
I got some feedback from Rob as we were troubleshooting a security_solution issue that this functionality needs to be wrapped in a `useEffect` otherwise we could get weird behavior (it lead to the security_solution not navigating to the cases app when it was supposed to).